tools/GlobalFilters.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GlobalFilters xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <ContentSearchFilters>
+    <ContentFilter>
+      <Name>Initial ContentFilters</Name>
+      <Filters>
```

Comment:
@kirthik Did we have anythign to do with this Filters section, or is this the default from the xml file?  If it's the default, I think we should only store the section with our specific changes.